src/zinolib/ritz.py

```diff
@@ -413,7 +417,8 @@ def _request(self, command: bytes, recv_buffer=4096, **_):
                     if header[0] >= 500:
                         # Die on Error codes
                         return DataResponse(header[1], header)
-                        # raise ProtocolError("Errorcode '%s %s' reported from server" % (header[0], header[1]))
+                        # raise ProtocolError("Errorcode '%s %s' reported from server"
+                        #  % (header[0], header[1]))
```

Comment:
`# noqa` or whatever prevents the reformatting is needed here. It's a commented out line, not a comment with more info.